processing/src/main/java/org/apache/druid/query/rowsandcols/semantic/DefaultFramedOnHeapAggregatable.java

```diff
@@ -454,6 +454,15 @@ private AppendableRowsAndColumns computeCumulativeAggregates(AggregatorFactory[]
       rowIdProvider.incrementAndGet();
     }
 
+    // Corner case when the numRows=upperOffset
+    // In such a case the priming of results is not needed
+    if (rowIdProvider.get() == numRows) {
+      for (int i = 0; i < aggs.length; i++) {
+        results[i][resultStorageIndex] = aggs[i].get();
```

Comment:
this should now be 
`results[i][resultStorageIndex] = aggFactories[i].finalizeComputation(aggs[i].get())`
since [this PR](https://github.com/apache/druid/pull/16244)

Similarly in reverse cummulative aggregate as well